Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -160,7 +160,7 @@ BTDiagnostics::DoDump (int step, int i_buffer, bool force_flush)
 
 
 bool
-BTDiagnostics::DoComputeAndPack (int step, bool force_flush)
+BTDiagnostics::DoComputeAndPack (int step, bool /*force_flush*/)
 {
     // always set to true for BTDiagnostics since back-transform buffers are potentially
```

Comment:
In this case, I would rather throw a warning if `force_flush` is set to something that is not implemented.

Reply:
Oh right, no in that specific case not. I should have expanded the body of the implementation, thanks for noticing!